src/shared/modules/connections/connectionsDuck.ts

```diff
@@ -151,6 +153,16 @@ export function getAuthEnabled(state: GlobalState): boolean {
   return data?.authEnabled ?? false
 }
 
+export function getHost(state: GlobalState): string | null {
```

Comment:
Agreed on the naming being a bit vague, but I kept it as just `host` since that's what it's called already and I was trying to limit the scope of the changes.